Packs/PAN-OS/Integrations/Panorama/Panorama_test.py

```diff
@@ -6380,3 +6380,55 @@ def test_second_fetch_with_two_incidents_with_different_log_types_and_different_
         assert last_id_dict.get('X_log_type', '') == '000000002'
         assert last_fetch_dict.get('Y_log_type', '') == '2022-01-01 13:00:00'
         assert last_id_dict.get('Y_log_type', '') == '000000002'
+
+
+case_only_name = (
```

Comment:
What will happen if you give both name and name_match?

Reply:
If there is an application that meets the requirements the API returns it else we get a warning entry for an empty list (this is a known issue related to pan-os response parsing)